sc-tools/sc-builder/tests/kb/test_templates/template_1.scs

```diff
@@ -0,0 +1,71 @@
+.._el_47459456
```

Comment:
pls either replace it with hand writen scs or add image for easier understanding of this template

Reply:
Added near with scs
![image](https://user-images.githubusercontent.com/56268020/205852151-1bcd2ac2-3805-4941-b2c8-aadca53e7b83.png)